static/app/views/issueList/actions/utils.tsx

```diff
@@ -148,3 +169,9 @@ export function getLabel(numIssues: number, allInQuerySelected: boolean) {
     return text + append;
   };
 }
+
+export function performanceIssuesSupportsIgnoreAction(
```

Comment:
I find this check a little weird, couldn't we just use `groupCapabilities` instead? Maybe I'm misunderstanding

Reply:
This takes the ignore options and checks if it uses the time window properties that performance issues don't support. I'd have to make that check whether or not we use groupCapabilities, but I suppose I could rename this to something like `ignoreActionHasTimeWindow()`

Reply:
Ah gotcha. Yeah the nomenclature confused me a bit but we're gonna get rid of it anyways when the feature is supported so this is fine